src/poliastro/plotting.py

```diff
@@ -241,205 +241,48 @@ def _plot_sphere(radius, color, name, center=[0, 0, 0] * u.km):
     return sphere
 
 
-class OrbitPlotter3D:
-    """OrbitPlotter3D class.
+class _BaseOrbitPlotter:
+    """
+    Parent Class for the 2D and 3D OrbitPlotter Classes based on Plotly.
     """
 
-    def __init__(self):
-        self._layout = Layout(
-            autosize=True,
-            scene=dict(
+    def __init__(self, dim3=True):
+        if dim3:
```

Comment:
Instead of this boolean parameter, why not override the `__init__` method in each subclass? You can leave the common stuff here, call `super().__init__()` in the subclasses and avoid having the `dim3` parameter.

Reply:
Somewhere in our discussions, we decided to use a flag. Okay I will remove it

Reply:
I recall I told you about a flag, but I did not predict it would be with this intent. See, I'm bad at discussing without seeing the code :)

Reply:
Oh! Sorry. I am just correcting things. Would commit the changes in a while. :)